common/plugins/eu.esdihumboldt.hale.common.test/Tests.product

```diff
@@ -203,6 +211,7 @@
       <plugin id="org.apache.commons.io"/>
       <plugin id="org.apache.commons.jxpath"/>
       <plugin id="org.apache.commons.lang"/>
+      <plugin id="org.apache.commons.logging"/>
```

Comment:
The only plugin that needs to be added here is `eu.esdihumboldt.cst.functions.geometric.test`. It does not introduce any new dependencies, so the other additions should be removed.

The neat option in Eclipse to *Add required plugins* is a bit of a pitfall here, as it usually will add plugins and fragments that are not desired - as with `org.apache.commons.logging`, which adds a second Apache Commons Logging implementation in addition to the existing one.